types/markdown-to-jsx/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-null-undefined-union": false
```

Comment:
This rule shouldn't be causing any errors, since it shouldn't be enabled for `dtslint`. Do you get any errors when running `dtslint` by invoking `npm run lint markdown-to-jsx` from the root of the `DefinitelyTyped` repo?

Reply:
React.ReactNode and multiple other types from React include both null and undefined. I could disable the rule on individual lines instead if that's better? (the error occurs in five places currently)

Reply:
Works for me. I think you might be running with a different version of TSLint locally than what we expect, and as a result you're getting errors from that rule. Doesn't hurt to disable the rule, though, so this will work fine.